node/pkg/p2p/p2p.go

```diff
@@ -268,6 +268,12 @@ func Run(
 
 		logger := supervisor.Logger(ctx)
 
+		var coerr error
```

Comment:
upon further review, I'm fairly confident that the bootstrap peer replacement code will not adequately catch all p2p use cases. perhaps it should be moved to `NewHost` instead. as written, this cutover check will not be performed for `cmd/ccq` or `hack/query/test` (which would suddenly break CI after the cutover) as they do not use `p2p.Run` and therefore would not hit this code path.